### PR TITLE
Histogram

### PR DESCRIPTION
**Pull request by aromatic-toast**
fix #8 

TODO:
-  needs more thorough tests in next milestone

- need to check for more edge cases during milestone 3

### CODE REVIEW

File: tests/test_hist.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import pytest
+import numpy as np
+from altairexpress import altairexpress as ae
+from gapminder import gapminder
+
+# define testing data
+gapminder
+
+
+def test_hist(gapminder):
```
Comment (jacktan1) [inline tests/test_hist.py:10]: Don't think test function should take inputs, just load the gap minder inside the function

File: tests/test_hist.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    # make histogram of input variable
+    plot = hist(data=gapminder, variable='lifeExp')
```
Comment (jacktan1) [inline tests/test_hist.py:12]: think this is supposed to be ae.hist?

**Review by jacktan1 — APPROVED** [review]
